### PR TITLE
Cherry-pick #9179 to 6.x: Updating elasticsearch/ccr metricset with ES ccr_stats API changes

### PR DESCRIPTION
**Pull request by ycombinator**
Cherry-pick of PR #9179 to 6.x branch. Original message: 

Resolves https://github.com/elastic/kibana/issues/25778.

The ES CCR API changed a little before the 6.5.0 FF and we didn't catch these changes 😞. This PR updates the CCR metricset code to work with the new ES CCR API response structure.

## Testing this PR
1. Setup CCR: https://www.elastic.co/guide/en/elastic-stack-overview/current/ccr-getting-started.html
2. Checkout this PR and `make` Metricbeat: `cd metricbeat; make`
3. Enable the `elasticsearch` module: `./metricbeat modules enable elasticsearch`
4. Edit `modules.d/elasticsearch.yml` and make sure the `ccr` metricset is enabled and `xpack.enabled: true` is set.
5. Start Metricbeat: `./metricbeat -e`
6. After about 10-20 seconds, check that Metricbeat indexed the correct document of `type` = `ccr_stats` into `.monitoring-es-mb-6-*`.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -55,6 +55,9 @@ https://github.com/elastic/beats/compare/v6.5.0...6.x[Check the HEAD diff]
 *Metricbeat*
 
 - Fix race condition when enriching events with kubernetes metadata. {issue}9055[9055] {issue}9067[9067]
+- Fix issue preventing diskio metrics collection for idle disks. {issue}9124[9124] {pull}9125[9125]
+- Fix panic on docker healthcheck collection on dockers without healthchecks. {pull}9171[9171]
+- Fix issue with not collecting Elasticsearch cross-cluster replication stats correctly. {pull}9179[9179]
```
Comment (ycombinator) [inline CHANGELOG.asciidoc]: Only this last entry should be part of this PR.

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
All green now. @jsoriano or @ruflin this backport PR is ready for review. Thanks!

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (ruflin) [reply CHANGELOG.asciidoc]: Can you clean it up?

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
jenkins, test this

**Pull request merged**